src/game/server/entities/character.cpp

```diff
@@ -1525,6 +1530,28 @@ void CCharacter::Tick()
 	//~ }
 	//~ else
 		//~ m_InWater = 0;
+	// Delayed Death
+	if(GetClass() == PLAYERCLASS_VOODOO && m_VoodooAboutToDie && m_VoodooTimeAlive > 0)
+	{
+		m_VoodooTimeAlive-=1000;
```

Comment:
sorry, i don't understand this. Why is it -1000? It will happen every tick, right?

Reply:
Yes, every tick. VoodooTimeAlive is in unit milliseconds for more fine tuning